08 - Fun with HTML5 Canvas/index-START.html

```diff
@@ -4,16 +4,67 @@
   <meta charset="UTF-8">
   <title>HTML5 Canvas</title>
 </head>
-<body>
-<canvas id="draw" width="800" height="800"></canvas>
-<script>
-</script>
+  <body>
+    <canvas id="draw" width="800" height="800"></canvas>
+    <script>
+      //// global variables
```

Comment:
in comparison this class looks much less structured. and look, you need fewer comments down here, because the class makes the structure clear.